src/components/Icon/index.js

```diff
@@ -36,7 +39,8 @@ const defaultProps = {
     fill: themeColors.icon,
     small: false,
     inline: false,
-    isSVGAvatar: false,
```

Comment:
qq, I remember we added this in the last PR, just to share some knowledge, why we removed it now?

Reply:
Rather than have the Icon component handle avatar styles, I decided that the Avatar component should own all things avatars. Therefore, I replaced the styling with additionalStyles and pass in the correct avatar styles from within the <Avatar> component instead.

Reply:
That said, I will remove isWorkspaceAvatar too!